tests/Api/Responses/Expected/admin/catalog_promotion/put_catalog_promotion_response.json

```diff
@@ -7,7 +7,7 @@
     "code": "mugs_discount",
     "startDate": null,
     "endDate": null,
-    "priority": 1000,
+    "priority": 0,
```

Comment:
Having one row in the database when updating, gedmo sets the priority to 0, even if the value given is different